src/locales/en/en.json

```diff
@@ -1595,7 +1595,8 @@
     "editidentifiererror": "Unable to update identifier",
     "failedupdateconnectionerror": "Failed to update connection",
     "duplicateconnectionerror": "Connection already established",
-    "groupidnotmatcherror": "Connection not part of this group"
+    "groupidnotmatcherror": "Connection not part of this group",
+    "failedaddappinitstate": "Failed to add app init state"
```

Comment:
Toasts imo should give either useful use feedback that they did something wrong, or if something unexpected goes wrong it should just be generic like "Something went wrong". There is the a `showGenericError` function it seems

@sdisalvo-crd Do you agree?

Reply:
Yes I agree. A case of usable feedback in a toast message could be when we scan a connection twice, while the `GenericError` (triggered by `showGenericError`) makes use of a pop up and often times has the intent to roll back the last action so the user can try again whatever went wrong.